addons/mail/models/mail_volume_setting.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class MailVolumeSetting(models.Model):
+    """ Represents the volume of the sound that the user of user_setting_id will receive from partner_id. """
+    _name = 'mail.volume.setting'
+    _description = 'Mail Volume Setting'
+
+    user_setting_id = fields.Many2one('mail.user.settings', required=True, ondelete='cascade')
+
+    partner_id = fields.Many2one('res.partner', required=True, ondelete='cascade')
+    volume = fields.Float(default=1.0, help="Ranges between 0.0 and 1.0, scale depends on the browser implementation")
```

Comment:
Actually why not storing it directly into settings ? Not sure to see the advantage of adding yet another table ...

Reply:
This one is a cross tab for the customized volume each user can set for independently for each other user.

Reply:
Ow, like "Tde" sets volume for "Seb" to 4 ans for "Al" to 0 ? Wow, impressive discord-copy-paste :D . Well, ok. At least it is stored in a separate table, I suppose it makes the job. Maybe rename to something like ``res.users.settings.volume`` then ?